README.md

```diff
@@ -27,6 +27,26 @@
   </a>
 </p>
 
+## Roadmap
+
+As of version 6.0, the Sensu Go web interface is no longer included in the sensu-go codebase. Users who download and compile Sensu Go from source will need to download and run the web interface as a separate component. Also, although they currently share certain features, the web interface included in the commercial distribution (see: https://sensu.io/downloads) and this project will no longer share a common codebase.
+
+### Goals
+
+The primary goal of this project is to offer a simple web interface that provides visibility into Sensu Go's "stateful" data (e.g., real-time invetory and real-time event dashboards). At this time, this includes the following:
+
+* Basic auth (login using Sensu Go user credentials)
+* Namespace switching
+* Views reflecting event, entity, and silence state. (Scoped by namespace.)
+
+### Non-Goals
+
+This project does not aim to provide a web-based interface for every feature of the open-source Sensu Go platform, specifically include the configuration of Sensu Go pipelines. As a result, certain features of the Sensu Go open-source web interface project may be marked as "deprecated" and/or removed in a future release (e.g., if changes to sensu-go breaks compatibility.)
```

Comment:
I would rephrase the second sentence here to say "As a result, certain features of this project may break with new releases of Sensu Go." (period), and then elaborate to suggest that "The Sensu Engineering team will review and merge PRs submitted by the community, and/or periodically publish new releases to mark certain features as "deprecated" (or just remove them)." 

Feel free to rephrase, but I think the true state of this project is that it's no longer actively maintained by the Sensu engineering team, so things may break and there is no SLA for fixes.